wpengine/atlas-content-modeler
# BH-1075 Allow creation of taxonomies in the developer app

- Adds a new taxonomies view at `/wp-admin/?page=atlas-content-modeler&view=taxonomies`.
- Adds a “View Taxonomies” button on the main model listings page (for now; this may move later).

<img width="1470" alt="Screenshot 2021-06-30 at 11 43 32" src="https://user-images.githubusercontent.com/647669/123939499-78ceb080-d998-11eb-809e-f672ed98d13b.png">

<img width="1452" alt="Screenshot 2021-06-30 at 11 43 21" src="https://user-images.githubusercontent.com/647669/123939512-7c623780-d998-11eb-98eb-f1a500bcc97c.png">

### To test
1. Visit the developer app at `/wp-admin/admin.php?page=atlas-content-modeler`.
2. Click “View Taxonomies”.
3. Try adding taxonomies. You should see them in the list to the right of the taxonomies form.
4. Run `wp option get atlas_content_modeler_taxonomies` or view that row in the WP options table to see the result of your submission as data.
 
### Notes
- We plan to improve the listing of taxonomies in [BH-1078](https://wpengine.atlassian.net/browse/BH-1078) (“List Taxonomies”). I added the listing area to the right of the form as a placeholder for now just to wire up the data.
- Perhaps we could re-use the existing form in [BH-1076](https://wpengine.atlassian.net/browse/BH-1076) (“Edit Taxonomy”)? We could hide the list view in edit mode and present only the form with submit/cancel buttons.
